recipes/pynumero_build/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+cd third_party/ASL
+./getASL.sh
+cd solvers
+./configurehere
+find . -name "makefile" -exec sed -i "s/CFLAGS = -DNo_dtoa -fPIC -O/CFLAGS = -fPIC -O/g" {} \;
+make
+cd ../../../
+
+mkdir build
+cd build
+echo "LOOKING FOR ampl-mp"
+find /opt/conda/pkgs -type d -name "ampl-mp*"
```

Comment:
I think you have to modify this part. Just put ``pkg-config`` as a build dependency and use it directly instead of doing that

Reply:
I am thinking on writing a cmake.find module in the future. The project uses that directory to link our compiled libraries to a library in ampl-mp. I am new to conda-forge so I am not sure if thats a robust way of doing it but for now seems to work for me.